docs/source/api/plugin/landing-pages.md

```diff
@@ -229,9 +259,131 @@ A boolean used to set whether Studio Explorer should include cookies in its Grap
 
 If you omit this, the Explorer defaults `includeCookies` to `false` or the current user setting.
 
+If you have `embed: true`, cookies will never be sent from your embedded Explorer.
+
+</td>
+</tr>
+
+</tbody>
+</table>
+
+### `embed` options
+
+These are the fields you can include in the `embed` option you pass to the ApolloServerPluginLandingPageProductionDefault:
+
+<table class="field-table api-ref">
+  <thead>
+    <tr>
+      <th>Name /<br/>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+
+<tr>
+<td>
+
+##### `displayOptions`
+
+`Object`
+
+</td>
+<td>
+
+An object containing additional display options related to the visual state of the embedded Explorer on page load.
+
+For supported subfields, see [`displayOptions` options](#displayoptions-options).
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+###### `persistExplorerState`
+
+`boolean`
+</td>
+<td>
+
+If `true`, the embedded Explorer uses `localStorage` to persist its state (including operations, tabs, variables, and headers) between user sessions. This state is automatically populated in the Explorer on page load.
+
+If `false`, the embedded Explorer loads with an example query based on your schema (unless you provide [`document`](#document)).
+
+The default value is `false`.
+
+</td>
+</tr>
+
+</tbody>
+</table>
+
+
+
+### `displayOptions` options
```

Comment:
Maybe `embed.displayOptions`? Certainly either the header or the next line should be explicit that this is nested under `embed`